File: area.cpp
#include <iostream>
using namespace std;
int area(int length, int width) {
    return length * width;    
}
double area(double length, double width) {
    return length * width;
}
int main() {
    cout << area(5, 10) << endl;          // Calls int version
    cout << area(5.5, 10.2) << endl;      // Calls double version
}

File: area.java
class Rectangle {
    static int area(int width, int height) {
        return width * height;
    }
    static double area(double width, double height) {
        return width * height;
    }
    public static void main(String[] args){
        System.out.println("Area of Rectangle with integer dimensions: " + area(5, 10));
        System.out.println("Area of Rectangle with double dimensions: " + area(5.5, 10.5));
    }   
}